rts/lra/coordinator/pom.xml

```diff
@@ -163,6 +163,7 @@
                                 <lra.coordinator.host>${lra.coordinator.host}</lra.coordinator.host>
                                 <lra.coordinator.url>${lra.coordinator.url}</lra.coordinator.url>
                                 <server.jvm.args>${server.jvm.args} -Dlra.coordinator.url=${lra.coordinator.url}</server.jvm.args>
+                                <lra.tck.timeout.factor>${lra.tck.timeout.factor}</lra.tck.timeout.factor>
```

Comment:
maybe it shouldn't be named with `tck` in the name.

Reply:
That's a good point. I was thinking if it's good to use it with the name `tck` in it but then I leave it as it is as it's how it's used.
I feel it's not a great approach. I will try to come with a general name.

Reply:
@xstefank I changed the naming. If you have a while, please, take a look.